spec/models/state_file_md_intake_spec.rb

```diff
@@ -140,12 +140,13 @@
       it "clears other account fields" do
         expect {
           intake.update(payment_or_deposit_type: "mail")
-        }.to change(intake.reload, :account_type).to("unfilled")
-          .and change(intake.reload, :bank_name).to(nil)
-          .and change(intake.reload, :routing_number).to(nil).and change(intake.reload, :account_number).to(nil)
-          .and change(intake.reload, :withdraw_amount).to(nil)
-          .and change(intake.reload, :date_electronic_withdrawal).to(nil)
-          .and change(intake.reload, :account_holder_name).to(nil)
+        }.to(change(intake.reload, :account_type)).to("unfilled")
+         .and(change(intake.reload, :bank_name)).to(nil)
+         .and(change(intake.reload, :routing_number)).to(nil)
+         .and(change(intake.reload, :account_number)).to(nil)
+         .and(change(intake.reload, :withdraw_amount)).to(nil)
+         .and(change(intake.reload, :date_electronic_withdrawal)).to(nil)
+         .and(change(intake.reload, :account_holder_name)).to(nil)
```

Comment:
added these parens because when i auto-formatted in rubymine the indentation got really f'd up